distribution/std-lib/Test/src/Test.enso

```diff
@@ -48,6 +48,22 @@ Any.should verb argument = Verbs.verb this argument
 ## Fail a test with the given message.
 fail message = Panic.throw (Failure message)
 
+## Expect a function to fail with the provided error.
+expect_fail_with ~action matcher =
+    res = Panic.recover action
+    case res of
+        _ -> here.fail "no exception"
```

Comment:
The style of this messages is horrible. Should be something like "Expected a panic to be thrown, but action succeeded."

Reply:
These are displayed in the final report!